tests/templates/kuttl/kerberos/secretclass.yaml

```diff
@@ -9,7 +9,9 @@ spec:
     kerberosKeytab:
       realmName: CLUSTER.LOCAL
       kdc: krb5-kdc
-      adminServer: krb5-kdc
+      admin:
+        mit:
+          adminServer: krb5-kdc
```

Comment:
Do we want to simply call it `server` as the term `admin` is already defined above?

Reply:
Resolved in #257